docs/doxygen/pages.dox

```diff
@@ -232,6 +234,21 @@ struct NetworkContext {
     // Fields necessary for the transport implementations, e.g. a TCP socket descriptor.
 };
 @endcode
+
+@section http_porting_time Time Function
+@brief The HTTP library optionally relies on a function to generate millisecond
+timestamps, for the purpose of calculating the elapsed time between non-zero
```

Comment:
You mentioned greater than zero elsewhere. However, I think both statements no longer apply when the first send or receive is a 0, and a timeout has already occurred.
```suggestion
timestamps, for the purpose of calculating elapsed time when the amount of data read is non-negative.
```

Reply:
Thanks